examples/js/postprocessing/OutlinePass.js

```diff
@@ -303,23 +302,16 @@ THREE.OutlinePass.prototype = Object.assign( Object.create( THREE.Pass.prototype
 			renderer.clear();
 			this.fsQuad.render( renderer );
 
-			this.tempPulseColor1.copy( this.visibleEdgeColor );
-			this.tempPulseColor2.copy( this.hiddenEdgeColor );
-
 			if ( this.pulsePeriod > 0 ) {
-
-				var scalar = ( 1 + 0.25 ) / 2 + Math.cos( performance.now() * 0.01 / this.pulsePeriod ) * ( 1.0 - 0.25 ) / 2;
-				this.tempPulseColor1.multiplyScalar( scalar );
-				this.tempPulseColor2.multiplyScalar( scalar );
-
+				this.pulseWeight = scalar;
```

Comment:
It seems `scalar` is not defined. I guess it's wrong to delete:
```js
var scalar = ( 1 + 0.25 ) / 2 + Math.cos( performance.now() * 0.01 / this.pulsePeriod ) * ( 1.0 - 0.25 ) / 2;
```

Reply:
@fiesh Any thoughts about this ?

Reply:
Hmm I thought I had fixed this before... I guess something must have gone wrong, will rebase again.